packages/bpk-component-tooltip/stories.js

```diff
@@ -49,17 +49,8 @@ storiesOf('bpk-component-tooltip', module)
     <div style={st}>
       <BpkTooltip
         id="my-tooltip"
-        target={<Heading level="h1">£295</Heading>}
-        tetherOptions={{
-          attachment: 'middle left',
-          constraints: [
-            {
-              to: 'window',
-              attachment: 'together',
-              pin: true,
-            },
-          ],
-        }}
+        target={<Heading>£295</Heading>}
```

Comment:
Why no more `level="h1", are we falling back to a default prop?

Reply:
The h1 ws redundant  - it was causing a react error of unknown prop on element

https://github.com/Skyscanner/backpack/blob/3a93a85ec6dedc485e0028399a8b6a1893dc6a8e/packages/bpk-component-tooltip/stories.js#L35